lib/src/models/generated_w3_header.dart

```diff
@@ -0,0 +1,24 @@
+class GeneratedW3Header {
```

Comment:
Let's use the full name `W3C*` instead of `W3*` for all the relevant components in the file.